webpack/components/AnsibleHostDetail/components/JobsTab/JobsTabHelper.js

```diff
@@ -77,3 +77,19 @@ export const useCancelMutation = (resourceName, resourceId) =>
       },
     ],
   });
+
+export const readableCron = (cron = '') => {
+  if (cron.match(/(\d+ \d+ \* \* \*)/)) {
+    return 'daily';
+  }
+
+  if (cron.match(/(\d+ \d+ \* \* \d+)/)) {
+    return 'weekly';
+  }
+
+  if (cron.match(/(\d+ \d+ \d+ \* \*)/)) {
+    return 'monthly';
+  }
+
+  return 'hourly';
```

Comment:
should we explicit here to "\d+ * * * *" and be default something like "custom"?